features/src/main/feature/spring-features.xml

```diff
@@ -38,6 +38,13 @@
         <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-tx/${spring.version}_1</bundle>
     </feature>
 
+    <feature name="spring-orm" version="${spring.version}">
+        <feature version='${spring.version}'>spring-tx</feature>
+        <feature version='${spring.version}'>spring-jdbc</feature>
```

Comment:
https://central.sonatype.com/artifact/org.springframework/spring-orm/6.0.12
`spring-jdbc` is needed with compile scope

Reply:
I would like to remain consistent with the rest of the spring features. As you can see, the other features don't contain dependencies of the SMX bundle like in Karaf https://github.com/apache/karaf/blob/main/assemblies/features/spring/src/main/feature/feature.xml#L72-L75.

Let's keep it like in Karaf if you don't mind

Reply:
But this makes me realize that we are not fully in sync with Karaf, at some point, we will need to update the spring feature file to be in sync, I will do that

Reply:
I'm not sure if I understood it right, could you take a look at my changes? 🙏

Reply:
You understood it perfectly right